### PR TITLE
Hamzabohra patch 2

### LINKED ISSUE #70: relational-operators C tutorial
**Description of the tutorial**
i'd like to create a tutorial on the use of relational operators in C

**Do you want to be assigned to work on this?**
(mark [x])
- [x] YES
- [ ] NO

**Optional: Any additional info you want to add**
...

### PR DESCRIPTION
**Pull request by hamzabohra**
### 2. Description
 This tutorial is a very beginner friendly introduction to relational operators in c language. The code is very beginner friendly with easy to understand comments. At the end there is a simple challenge is well

### 3. Fill in checklist by marking [x]

- [x] I've read the CONTRIBUTING.md
- [x] I was assigned to this issue
- [x] My code is formatted using required linting
- [x] I've run my code locally and checked it works

### CODE REVIEW

**Comment by hamzabohra** [comment]
thnx for letting me contribute.

File: 05-operators/relational-operators.c [hunk 1]
```diff
@@ -1,15 +1,71 @@
...
 ------------------------------------------------------------------------------------
- Tutorial: brief description of tutorial content
+ Tutorial: Some of the operators that are frequently used in loops or if conditions are the relational operators. 
+           To examine the relationship between the two variables, relational operators are frequently employed in C. 
+           It will return value 1 if the relation is true. If not, it returns a value of 0.
+           
+           ------------------------------------------------------------------------------------------
+           | Operator |    Usage    |          Description               |         Example          |
+           ------------------------------------------------------------------------------------------
+           |  ==      |     x==y    |    x is equals to y                |  10==9 returns true [1]  |
```
Comment (funbeedev) [inline 05-operators/relational-operators.c:10]: 10==9 should return false. Please modify

**Review by funbeedev — CHANGES_REQUESTED** [review]
One minor correction to be made, thanks!

**Comment by hamzabohra** [comment]
i have made changes. please check it.
:)

**Review by funbeedev — APPROVED** [review]

**Comment by funbeedev** [comment]
@hamzabohra  looks good, thanks again :)

**Pull request merged**